### DISCUSSION TITLE
Why do you use a git force push in your CI and CI examples?

### DISCUSSION THREAD

**Discussion by hcharley** [jscutlery/semver#481] [Q&A] [answered]
I am wondering why the `--force` flag is used in your Github workflow example.

**Comment by edbzn** [comment, marked as answer]
Hi @hcharley, it's because we are re-tagging `last-release` after each release to use it as `base` for the affected command. Because we override the tag we have to push force.

**Reply by Bielik20** [reply]
I just wonder. Have you tried using https://github.com/nrwl/nx-set-shas?
I am considering moving from my semantic-release setup to your library, and this is something I would probably use. I haven't tested it though, but according to docs it retrieves hash of last successful run and uses that as a base.

https://github.com/Bielik20/ts-libs/blob/28c5dff4e0aa322f0831b52546488e29be5904d3/.github/workflows/release.yml#L22-L25
